src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -80,6 +80,9 @@
    :up-time-ms WholeMilliseconds
    :start-time-ms WholeMilliseconds})
 
+(def DebugLoggingConfig
+  (schema/maybe {:interval-milliseconds schema/Int}))
```

Comment:
I changed the name because the value was in milliseconds before, it was just mislabeled. 

Yeah, agreed that the user wouldn't want milliseconds. We could change it to minutes, and just convert it to milliseconds for the scheduler service. Tried putting a float in, it works fine